samples/https/nginx-app.yaml

```diff
@@ -1,44 +0,0 @@
-apiVersion: v1
```

Comment:
should we remove the whole folder? The other file was just a config file for this sample

Reply:
I agree that we should remove the folder if we are removing the file. Looking at istio.io, these files are called out most recently in the 1.5 archive (when referring to the English docs (the Chinese docs which are behind have it in their current docs. (discussion to not display translated docs more than a yr old in the future))).

Reply:
The new translation model we're talking about moving to will automatically redirect to the english version of any doc if the translated version is out of date relative to the english one.

Reply:
Remove them.